packages/adapter/addon/-private/utils/fetch.ts

```diff
@@ -2,19 +2,23 @@ import require, { has } from 'require';
 
 type FetchFunction = (input: RequestInfo, init?: RequestInit | undefined) => Promise<Response>;
 
-let _fetch: (() => FetchFunction) | null = null;
+let _fetch: FetchFunction | null = null;
```

Comment:
Actually, we can't cache this here. If we do, I believe that we break the ability for pretender to patch it.

Reply:
fixed